src/main/java/org/commcare/util/FormMetaIndicatorUtil.java

```diff
@@ -0,0 +1,24 @@
+package org.commcare.util;
+
+import org.javarosa.core.model.FormDef;
+import org.javarosa.core.model.instance.TreeReference;
+
+import io.reactivex.annotations.Nullable;
+
+/**
+ * @author $|-|!˅@M
+ */
+public class FormMetaIndicatorUtil {
+
+    public static final String CASE_NAME_DESCRIPTOR = "Pragma-Case-Name";
```

Comment:
I think this should be titled around the output of the descriptor, not the most common input. The summary descriptor here can reference any data from within the form or without, so it might cause confusion to imply that its function is specific to cases.